app/scripts/modules/core/src/application/modal/PermissionsConfigurer.tsx

```diff
@@ -133,6 +133,12 @@ export class PermissionsConfigurer extends React.Component<IPermissionsConfigure
     }
   }
 
+  private willApplicationLockoutAllUsers(): boolean {
+    return !!(this.props.permissions) &&
+        !!(compact(this.props.permissions.READ).length) &&
```

Comment:
Would be more consistent with the logic in the modal controllers (and you wouldn't have to coerce 0 into `false`) if you used `compact(this.props.permissions.READ).length > 0`. Not a big deal.